docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -254,13 +254,31 @@ properties:
   * - ``iceberg.nessie-catalog.default-warehouse-dir``
```

Comment:
Let's leave out the following statement from the commit comment:

> To reduce the review effort, only the basic Nessie configurations were supported in https://github.com/trinodb/trino/pull/11701.

It doesn't bring any extra value to the current commit.

Reply:
removed.